README.md

```diff
@@ -222,6 +257,44 @@ result = tokenizer.instruct_tokenizer.tokenizer.decode(out_tokens[0])
 print(result)
 ```
 
+- *Multimodal Instruction Following*:
+
+
+```python
+from pathlib import Path
+
+from huggingface_hub import snapshot_download
+from mistral_common.protocol.instruct.messages import ImageURLChunk, TextChunk
+from mistral_common.tokens.tokenizers.mistral import MistralTokenizer
+from mistral_inference.generate import generate
+from mistral_inference.transformer import Transformer
+
+model_path = Path.home().joinpath("mistral_models") / "mistral-small-3.1-instruct" # change to extracted model
+
+tokenizer = MistralTokenizer.from_file(model_path / "tekken.json")
+model = Transformer.from_folder(model_path)
+
+url = "https://huggingface.co/datasets/patrickvonplaten/random_img/resolve/main/yosemite.png"
+prompt = "The above image presents an image of which park ? Please give the hints to identify the park."
+
+user_content = [ImageURLChunk(image_url=url), TextChunk(text=prompt)]
+
+tokenizer_output = tokenizer.instruct_tokenizer.encode_user_content(user_content, False)
+
+out_tokens, _ = generate(
+    [tokenizer_output.tokens],
```

Comment:
Think we need to test with public [mistral_common](https://github.com/mistralai/mistral-common/releases/tag/v1.5.4) which returns a tuple I believe no?